service/gradle/testing.gradle

```diff
@@ -158,4 +158,17 @@ task awsConnectedTest(type: Test) {
   outputs.upToDateWhen { false }
   finalizedBy tasks.combinedJaCoCoReport
   maxParallelForks = runnerThreads
-}
\ No newline at end of file
+}
+
+// contract tests
+test {
```

Comment:
I haven't tried to run these myself, but I assume it's fine to have multiple `test` blocks in the same file? Would it be possible to move all of this into the `test` block up above?

Reply:
yes, you can have an arbitrary number - you just use as many blocks as you need for your organization.
I was just trying to keep the contract testing separate, under the assumption it would be easier to understand.

I changed them to the same pattern as the other tests in the file.
The definition styles are equivalent, but stand out a bit less